packages/react/src/common/useFocusByKeyboard.tsx

```diff
@@ -24,21 +24,18 @@ const FOCUSABLE_ELEMENTS = [
   'select:not([disabled])',
 ]
 
-type FocusWithArrowsOptions = {
-  ref: RefObject<HTMLDivElement>
```

Comment:
Why move ref from the params object? Adding it to the object documents it nicely, imo